test/e2e/kubectl.go

```diff
@@ -298,7 +298,11 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 
 			// Build the static kubectl
 			By("Finding a static kubectl for upload")
-			testStaticKubectlPath, err := findBinary("kubectl", "linux/386")
+			targetBinary := "linux/386"
```

Comment:
To detect os/arch is as simple as `fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH)`
I don't see a case when we should use 386 instead of amd64